src/Sylius/Bundle/CoreBundle/Sitemap/Renderer/SitemapRenderer.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+ 
+namespace Sylius\Bundle\CoreBundle\Sitemap\Renderer;
+
+use Sylius\Bundle\CoreBundle\Sitemap\Model\SitemapInterface;
+
+/**
+ * @author Arkadiusz Krakowiak <arkadiusz.krakowiak@lakion.com>
+ */
+class SitemapRenderer implements SitemapRendererInterface
+{
+    /**
+     * @var RendererAdapterInterface
+     */
+    private $adapter;
+
+    /**
+     * @var array
+     */
+    private $configuration;
+
+    /**
+     * @param RendererAdapterInterface $adapter
+     * @param array $configuration
+     */
+    public function __construct(RendererAdapterInterface $adapter, array $configuration = array())
+    {
+        $this->adapter = $adapter;
+        $this->configuration = $configuration;
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function render(SitemapInterface $sitemap)
+    {
+        $this->adapter->setTemplate($this->configuration['template']);
```

Comment:
This makes the adapter stateful, why not passing the configuration to the Adapter's render method in order to have easy way to add and use more configuration parameter, what do you think?